.github/workflows/dart.yml

```
@@ -60,7 +60,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, macos-latest, windows-latest]
-        sdk: [3.2.0, dev]
+        sdk: [3.5.0, dev]
```

```suggestion
        sdk: [3.5, dev]
```

Then it'll pick up all of the "dot" releases automatically, which  is nice